api/rest/property_value.md

```diff
@@ -6,156 +6,492 @@ parent: REST
+ - `dcids`: Data Commons uniquely identifies nodes by assigning them DCIDs, or Data Commons IDs. Your query will need to specify the DCIDs for the nodes of interest.
+ - `property`: The property whose value you are interested in.
 
-    ```bash
-    curl 'https://api.datacommons.org/node/property-values?dcids=geoId/05&property=location&valueType=Election&limit=5'
-    ```
+In addition to these required properties, this endpoint also allows four optional arguments.
 
-*   Query with direction
+  - `valueType`: If the property queried only takes on node values, you can use this argument to filter nodes in the response, ensuring the response only contains nodes with the specified type.
 
-    ```bash
-    curl 'https://api.datacommons.org/node/property-values?dcids=geoId/05&property=containedInPlace&direction=out'
-    ```
+  - `direction`: You can specify this argument as `out` to indicate that you desire the response to only include nodes which are supercategories of the specified `DCIDs`, or `in` to only return nodes that are subcategories of the specified `DCIDs`. (For example, South America is a supercategory of Argentina, which in turn is a supercategory of Buenos Aires.)
+  
+  - `limit`: (≤ 500) Maximum number of values returned per node.
 
-## POST Request
+  - `key`: Your API key.
 
-**Examples**
+### Step 2: creating the request
 
-```bash
-curl -X POST 'https://api.datacommons.org/node/property-values' \
--d '{"dcids": ["geoId/05", "geoId/06"], "property": "name"}'
-```
+When actually putting together your request, you can choose from two options. If you intend to use only a small number of parameters, you may want to use the simpler formatting offered by the GET method, which makes requests against the main endpoint while altering the query parameters incorporated into the URL. For more complex queries, or if you prefer to utilize a static URL, a POST request likely makes more sense. To use it, make a POST request against the main endpoint while changing the fields of the JSON body it sends.
 
-<iframe width="100%" height="300" src="//jsfiddle.net/datacommonsorg/285dcqb4/12/embedded/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
+## What to expect in the response
 
-## Success Response
-
-### **Code**: `200 OK`
-
-**Response content example**
+Your response will always look like this:
 
 ```json
 {
     "payload": "<payload string>",
 }
 ```
 
-The equivalent JSON of "payload string" is as follows.
+Here `"<payload string>"` is replaced by JSON, whose structure changes depending on whether the response contains node references.
 
-**Example 1:** property values that are not nodes.
+**Structure 1:** Response for property values that are not node references.
 
 ```json
 {
-    "geoId/05": {
-        "out": [
-          {
-            "value": "Arkansas",
-            "provenanceId": "dc/sm3m2w3"
-          }
-        ]
-    },
-    "geoId/06": {
-        "out": [
+    "dcid": {
```

Comment:
I think we need to be very clear that this is what you get _after_ decoding the payload.

Reply:
I've added language to indicate the hard decoding requirement.